library.sh

```diff
@@ -794,11 +794,17 @@ function shellcheck_new_files() {
 }
 
 function latest_version() {
```

Comment:
Please add this comment and we can approve:

```
This function works "best effort" and works on Prow but not necessarily locally.
The problem is finding the latest release. If a release occurs on the same commit which
 was branched from master, then the tag will be an ancestor to any commit derived from master.
 That was the original logic. Additionally in a release branch, the tag is always an ancestor.
However, if the release commit ends up not the first commit from master, then the tag is not
 an ancestor of master, so we can't use `git describe` to find the most recent versioned tag. So
 we just sort all the tags and find the newest versioned one.
But when running locally, we cannot(?) know if the current branch is a fork of master or a fork
 of a release branch. That's where this function will malfunction when the last release did not
 occur on the first commit -- it will try to run the upgrade tests from an older version instead
 of the most recent release.
Workarounds include:
 Tag the first commit of the release branch. Say release-0.75 released v0.75.0 from the second commit
  Then tag the first commit in common between master and release-0.75 with `v0.75`.
 Always name your local fork master or main.
```

Reply:
Added as suggested.